src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -254,16 +261,16 @@ private Cci.DebugSourceDocument GetSourceDocument(CSharpSyntaxNode syntax, FileL
             return _debugDocumentProvider.Invoke(path, basePath: "");
         }
 
-        private BoundStatement AddAnalysisPoint(CSharpSyntaxNode syntaxForSpan, SyntheticBoundNodeFactory statementFactory)
+        private BoundStatement AddAnalysisPoint(CSharpSyntaxNode syntaxForSpan, Text.TextSpan? alternateStartSpan, SyntheticBoundNodeFactory statementFactory)
```

Comment:
If it's not complicated, I'd vote for passing the entire span, and perhaps as an optional parameter.